remote.go

```diff
@@ -694,6 +697,8 @@ func populateFetchOptions(options *C.git_fetch_options, opts *FetchOptions) {
 	options.custom_headers = C.git_strarray{}
 	options.custom_headers.count = C.size_t(len(opts.Headers))
 	options.custom_headers.strings = makeCStringsFromStrings(opts.Headers)
+	options.proxy_opts = C.git_proxy_options{}
```

Comment:
both `git_fetch_init_options()` and `populateProxyOptions()` take care of this, so it can be safely omitted.